Enable process metric collection added (#3304)
SumoLogic/sumologic-documentation: docs/integrations/hosts-operating-systems/opentelemetry/linux-opentelemetry.md

- `Messages`
- `yum.log`

:::note 

> By default, the path for linux log files required for all the distros are pre populated in the UI. Not all of the files might be available on your Linux distribution and unwanted file paths can be removed from the list. This is an optional step and the collection will work properly even if not all of the files are present on your system. If in doubt, you can leave the default file paths values.

Reviewer: ```suggestion
By default, the path for Linux log files required for all the distros are pre-populated in the UI. Not all of the files might be available on your Linux distribution. Optionally, you can remove unwanted file paths from the list. OpenTelemetry collection will still work properly even if not all of the files are present on your system. If in doubt, you can leave the default file paths values.
```

:::

#### Enable process metric collection (Optional)

> By default, the collector will not be sending process metrics to Sumo Logic this is because the number of processes running on a host can be very large, this may result in significant increase in Data Points per Minute (DPM).

Reviewer: ```suggestion
By default, the collector will not send process metrics to Sumo Logic; the number of processes running on a host can be very large, which would result in significant increase in Data Points per Minute (DPM).
```

- `Messages`
- `yum.log`

:::note 
By default, the path for Linux log files required for all the distros are pre-populated in the UI. Not all of the files might be available on your Linux distribution. Optionally, you can remove unwanted file paths from the list. OpenTelemetry collection will still work properly even if not all of the files are present on your system. If in doubt, you can leave the default file paths values.
:::

#### Enable process metric collection (Optional)

> By default, the collector will not send process metrics to Sumo Logic; the number of processes running on a host can be very large, which would result in significant increase in Data Points per Minute (DPM).

Reviewer: ```suggestion
By default, the collector will not send process metrics to Sumo Logic. This is because the number of processes running on a host can be very large, which would result in a significant increase in Data Points per Minute (DPM).
```

Click the **Enable process metric collection** checkbox to collect (process level metric)[https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/processscraper/documentation.md].
- **Name of process**. Add the list of process names.
- **Include/Exclude the above pattern**. Signifies if you want to exclude or include the metrics for the processes listed previously.
- **Match type for process name**. Select if the process name given should be considered for a strict match with the host machine processes or if it should be considered as regex when matching.

<img src='img/integrations/hosts-operating-systems/process-metric-collection.png' alt="process-metric-collection" width="400"/>

:::note 

> If the process list needs to be edited in future, you can edit it manually in the OTEL config yaml by adding/removing in the names list under process scrapper.

Reviewer: ```suggestion
If the process list needs to be edited in the future, you can edit it manually in the OTEL config yaml by adding/removing in the names list under process scrapper.
```
